src/devices/wholeBodyDynamics/SixAxisForceTorqueMeasureHelpers.cpp

```diff
@@ -32,13 +46,41 @@ const iDynTree::Wrench& SixAxisForceTorqueMeasureProcessor::offset() const
     return m_offset;
 }
 
+iDynTree::Wrench& SixAxisForceTorqueMeasureProcessor::estimatedOffset()
+{
+    return m_estimated_offset;
+}
+
+const iDynTree::Wrench& SixAxisForceTorqueMeasureProcessor::estimatedOffset() const
+{
+    return m_estimated_offset;
+}
+
+double& SixAxisForceTorqueMeasureProcessor::tempOffset()
+{
+    return m_tempOffset;
+}
+
+const double& SixAxisForceTorqueMeasureProcessor::tempOffset() const
+{
+    return m_tempOffset;
+}
+
 iDynTree::Wrench SixAxisForceTorqueMeasureProcessor::filt(const iDynTree::Wrench& input) const
 {
     Eigen::Matrix<double,6,1> retEig = toEigen(m_secondaryCalibrationMatrix)*toEigen(input)-toEigen(m_offset);
+    iDynTree::Wrench ret;
+    fromEigen(ret,retEig);
+    return ret;
+}
```

Comment:
As these is private code (their headers are not installed) if the variant of the methods **without temperature** is not used anymore, let's just remove them to simplify maintenance.

Reply:
Yes that seems to be the case, in the current code they are not used, should I directly eliminate them or do something like just send 0 in temperature so the behavior is the same while keeping the functionality even when calling the function without temperature?